docker-compose.yml

```diff
@@ -1,9 +1,36 @@
 version: '2'
 services:
+  fleet:
+    build:
+      context: .
+      dockerfile: Dockerfile.compose
+    image: kolide-local
+    command:
+      - /bin/bash
+      - "-c"
+      - "(
+          while ! grep -q 00:01BB /proc/net/tcp6;do echo wait;sleep 1;done; \
+          fleetctl config set --address https://localhost:443 --tls-skip-verify; \
+          fleetctl setup --email kolide@kolide.com --password kolide; \
+          fleetctl get enroll-secret > /data/enroll-secret;cp ./tools/osquery/kolide.crt /data/ \
+        ) & \
+        fleet prepare db && fleet serve"
+    environment:
+      KOLIDE_SERVER_ADDRESS: 0.0.0.0:443
+      KOLIDE_MYSQL_ADDRESS: mysql:3306
+      KOLIDE_MYSQL_PASSWORD: kolide
+      KOLIDE_REDIS_ADDRESS: redis:6379
+      KOLIDE_AUTH_JWT_KEY: E6VMWGBgZkKzfY/dw3zsZMd9foPsUrmL
```

Comment:
Is there any significance to this? If it's a random string, can it start "FAKESECRET" or something to be clear?

Reply:
Good point. I changed it to one that shows clearly it is a fake key.